src/background/api.ts

```diff
@@ -30,3 +38,20 @@ export const postTweet = async (toBeTweeted: ToBeTweeted, dispatchBackgroundActi
     return dispatchBackgroundActions.postTweetFailure(error)
   }
 }
+
+function makeHandleRequest(url: string): Promise<Response> {
+  const params = { domain: url }
+  const requestURL = new URL(`${window.roarServerUrl}/v1/website`)
+  requestURL.search = new URLSearchParams(params).toString()
```

Comment:
These `.toString()`s may be omitted, the roarServerUrl used as the base, and the `params` inlined to tidy this up a bit

```typescript
  const requestURL = new URL('v1/website', window.roarServerUrl})
  requestURL.search = new URLSearchParams({ domain: url })
  return fetch(requestURL)
```